src/components/org-users/org-users.njk

```diff
@@ -99,13 +113,16 @@
       </td>
       <td class="govuk-table__cell">
         {% set comma = joiner(", ") %}
```

Comment:
Do we still need this line, now we're not using `comma()` on 103?

Reply:
:done: